### ISSUE TITLE
Network

### ISSUE THREAD

**Issue by l3mpik** [Gwion101/Agar.io-Lite#2]
Network over socket.io for multiplayer games is bad idea ;/ Use clear Html5 WebSocket ;) btw. Nice work! ;)

**Comment by Gwion101** [comment]
Yes, I noticed there seems to be a lot of people arguing the case of WebSocket over Socket.io. I personally went with socket.io as I've had more experience with that module. I've mostly used Socket.io for work involving embedded systems so I used it here also. I am considering possibly refactoring the project and trying WebSockets at some point.
